dev/com.ibm.ws.cdi.internal/resources/com/ibm/ws/cdi/internal/resources/CDI.nlsprops

```diff
@@ -76,3 +76,15 @@ spi.extension.failed.to.construct.CWOWB1010E.useraction=Fix the error that is pr
 spi.extension.class.in.different.bundle.CWOWB1011E=CWOWB1011E: A {0} class registered through the CDI Extension SPI is in a different bundle than the CDIExtensionMetadata implementation.
 spi.extension.class.in.different.bundle.CWOWB1011E.explanation=The class is ignored because it was registered with the getExtensions() or getBeanClasses() method of a CDIExtensionMetadata implementation and is in a different bundle than the CDIExtensionMetadata implementation.
 spi.extension.class.in.different.bundle.CWOWB1011E.useraction=Move the affected class or create a new CDIExtensionMetadata implementation in the correct bundle.
+
+exception.creating.extensions.CWOWB1012E=CWOWB1012E: Exception creating extensions for the {0} archive. The error was: {1}
+exception.creating.extensions.CWOWB1012E.explanation=CDI failed to load or invoke the construction of an extension class
+exception.creating.extensions.CWOWB1012E.useraction=Fix the error that is preventing the extension from being created
+
+bce.not.loadable.CWOWB1013E=CWOWB1013E: The {0} BuildCompatibleExtension class cannot not be loaded
+bce.not.loadable.CWOWB1013E.explanation=The application includes a META-INF/services/jakarta.enterprise.inject.build.compatible.spi.BuildCompatibleExtension file which includes the class, but CDI could not load the class
+bce.not.loadable.CWOWB1013E.useraction=Ensure the class is present in the application and can be loaded. If the class is present in the application, there may be an FFDC file which has more information about why the class could not be loaded.
```

Comment:
there may be an FFDC file which has more information..
--->
an FFDC file might have more information...